app/views/shared/_size_and_zone_filters.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="dropdown show mb-8px mr-8px">
+  <a
+    id="dropdownMenuLink"
+    class="btn btn-sm btn-outline btn-secondary dropdown-toggle"
+    href="#"
+    role="button"
+    data-toggle="dropdown"
+    data-display="static"
+    aria-haspopup="true"
+    aria-expanded="false"
+  >
+    <% if @selected_zones.size == 1 %>
+      <%= @selected_zones.first.capitalize %>
+    <% else %>
+      All blocks
+    <% end %>
+  </a>
+  <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+    <form id="query-filters">
+      <button
+        class="dropdown-item <%= "active" unless @selected_zones.size == 1 %> filter-by-zone"
+        data-value=""
+      >
+        All blocks
+      </button>
+      <div class="dropdown-divider"></div>
+      <% @zones.each do |zone| %>
+        <button
+          class="dropdown-item query-filter-zone <%= "active" if @selected_zones.size == 1 && @selected_zones.first == zone %> filter-by-zone"
+          data-value="<%= zone %>"
```

Comment:
`my_facilities_filters.js` can also be removed in a similar fashion as above by making this:

```
<button
  class="dropdown-item query-filter-zone <%= "active" if @selected_zones.size == 1 && @selected_zones.first == zone %>"
  value="<%= zone %>"
  type="submit"
  name="zone"
  form="query-filters">
  <%= zone.capitalize %>
</button>
```